src/test/java/org/folio/rest/impl/InvoiceLinesApiTest.java

```diff
@@ -280,6 +286,37 @@ public void deleteInvoiceLinesBadLanguageTest() {
     verifyDeleteResponse(endpoint, TEXT_PLAIN, 400);
   }
 
+  @Test
+  public void fundValidationTest()
+  {
```

Comment:
The same with brackets formatting, use the same formatting as in other methods

Reply:
done